### DISCUSSION TITLE
播放功能建议增加默认清晰度选择1080P，和自动连播下一集不要播放推荐视频功能

### DISCUSSION THREAD

**Discussion by riverOcea** [the1812/Bilibili-Evolved#1704] [功能建议 / Ideas]
前几天开始发现B站网页版默认播放720P，每次都要手动点一下1080P，也不知道是不是陈叔叔没有钱了。
另外多分P的视频本来是会开启【自动连播】选项是没问题的，最近单集视频播放完会自动播放推荐的第一个视频，非常烦人，如果直接关掉【自动连播】选项，看新番之类的多P视频又无法自动播放。

**Comment by Geeyun-JY3** [comment]
前半是否确认可以复现？我这边没问题。

后半见 #1634

**Comment by Geeyun-JY3** [comment]
后半见：https://github.com/the1812/Bilibili-Evolved/discussions/1634#discussioncomment-593169
